### ISSUE TITLE
PWE Procedures section does not describe procedures

### ISSUE THREAD

**Issue by dbooth-boston** [w3c/PWETF#344]
I read the [Procedures section](https://www.w3.org/about/positive-work-environment/#Procedures), expecting it to tell me the steps one should take, if there is a Code violation.  But it only says: "escalation of the issue takes place by contacting one of the [W3C Ombuds](https://www.w3.org/about/positive-work-environment/#ombuds)".  That's all?  It feels like it should say more, especially if an Ombud has no authority to take action.

**Comment by rachelyager-nyc** [comment]
I agree that the Procedure Section needs more clarity on the documentation, reporting, and resolution tracking.

**Comment by TzviyaSiegman** [comment]
This page dates back to 2014. We will update it when we the ombuds and dispute resolution docs are finalized.
